### ISSUE TITLE
ITMS-90060 invalid bundle id

### ISSUE THREAD

**Issue by feinstruktur** [andreacremaschi/GEOSwift#10]
The geos dependency of 3.4.2a prevents submission to the app store. I’ll take a stab at fixing it and opening a PR.

![screen shot 2015-11-13 at 08 29 01](https://cloud.githubusercontent.com/assets/65520/11141888/46cd5904-89e1-11e5-991e-87ea3a7d1b81.png)

**Comment by andreacremaschi** [comment]
Ugh. Thank you for the heads up, what's your idea for the PR?

**Comment by feinstruktur** [comment]
I wish I had one =) I was thinking I’d go in and tweak the upstream version number but I’m not familiar enough with the cocoapods build structure.

I guess one could always run a script phase at the end in GEOSwift and strip out the character, like so
```
version="3.4.2a"
version=${version//[A-Za-z]}
info_plist=$BUILT_PRODUCTS_DIR/$PRODUCT_NAME.app/Info.plist
/usr/libexec/PlistBuddy -c "Set :CFBundleVersion $version" $info_plist
```
but it feels a bit hacky.

**Comment by andreacremaschi** [comment]
Ok, just change the value by hand before submitting. I am the mantainer of the geos pod and I was going to update it for the 3.5.0 anyway. I'll do it ASAP

**Comment by feinstruktur** [comment]
Ok, I'll got for that. Thanks!

**Comment by andreacremaschi** [comment]
fixed in release 0.3.1, closing
